language/examples/oss-samples/langchain/question_answering_with_large_documents_langchain.ipynb

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
<p>Should perhaps show more examples of Q&amp;A working in action with different questions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
<p>I was thinking to be consistent and comparing outputs of all methods with respect the same question</p>